app/tasks/batch.py

```diff
@@ -122,25 +122,15 @@ def submit_batch_job(
             "memory": job.memory,
             "environment": job.environment,
         },
-        "retryStrategy": {"attempts": job.attempts},
+        "retryStrategy": {
+            "attempts": job.attempts,
+            "evaluateOnExit": [{"onStatusReason": "Host EC2*", "action": "RETRY"}],
```

Comment:
Nice, this gives me the idea that we could utilize a different exit code (ie 2) in PixETL for when some of the tiles failed but not all. We could leverage this to trigger a retry. In that case we could add another `evaluateOnExit` with `onExitCode`

Reply:
Yup! You can add up to five retry conditions.